docs/ru/index.md

```diff
@@ -33,37 +33,37 @@ ClickHouse - столбцовая система управления базам
 | EventTime:  | 2016-05-18 05:19:20 | 2016-05-18 08:10:20 | 2016-05-18 07:38:00 | …   |
 
 В примерах изображён только порядок расположения данных.
-То есть, значения из разных столбцов хранятся отдельно, а данные одного столбца - вместе.
+То есть значения из разных столбцов хранятся отдельно, а данные одного столбца — вместе.
 
 Примеры столбцовых СУБД: Vertica, Paraccel (Actian Matrix, Amazon Redshift), Sybase IQ, Exasol, Infobright, InfiniDB, MonetDB (VectorWise, Actian Vector), LucidDB, SAP HANA, Google Dremel, Google PowerDrill, Druid, kdb+.
 {: .grey }
 
 Разный порядок хранения данных лучше подходит для разных сценариев работы.
-Сценарий работы с данными - это то, какие производятся запросы, как часто и в каком соотношении; сколько читается данных на запросы каждого вида - строк, столбцов, байт; как соотносятся чтения и обновления данных; какой рабочий размер данных и насколько локально он используется; используются ли транзакции и с какой изолированностью; какие требования к дублированию данных и логической целостности; требования к задержкам на выполнение и пропускной способности запросов каждого вида и т. п.
+Сценарий работы с данными — это то, какие производятся запросы, как часто и в каком соотношении; сколько читается данных на запросы каждого вида — строк, столбцов, байтов; как соотносятся чтения и обновления данных; какой рабочий размер данных и насколько локально он используется; используются ли транзакции и с какой изолированностью; какие требования к дублированию данных и логической целостности; требования к задержкам на выполнение и пропускной способности запросов каждого вида и т. п.
 
 Чем больше нагрузка на систему, тем более важной становится специализация под сценарий работы, и тем более конкретной становится эта специализация. Не существует системы, одинаково хорошо подходящей под существенно различные сценарии работы. Если система подходит под широкое множество сценариев работы, то при достаточно большой нагрузке, система будет справляться со всеми сценариями работы плохо, или справляться хорошо только с одним из сценариев работы.
 
-## Ключевые особенности OLAP сценария работы {#kliuchevye-osobennosti-olap-stsenariia-raboty}
+## Ключевые особенности OLAP-сценария работы {#kliuchevye-osobennosti-olap-stsenariia-raboty}
 
--   подавляющее большинство запросов - на чтение;
+-   подавляющее большинство запросов — на чтение;
 -   данные обновляются достаточно большими пачками (\> 1000 строк), а не по одной строке, или не обновляются вообще;
 -   данные добавляются в БД, но не изменяются;
--   при чтении, вынимается достаточно большое количество строк из БД, но только небольшое подмножество столбцов;
--   таблицы являются «широкими», то есть, содержат большое количество столбцов;
+-   при чтении «вынимается» достаточно большое количество строк из БД, но только небольшое подмножество столбцов;
```

Comment:
I don't understand this change.

Reply:
Dashes instead of hyphens, hyphens instead of spaces in compounds, i. e. only typographic and punctuation fixes